tests/runtests.py

```diff
@@ -231,7 +249,8 @@ def test_cmd_heap_bins_fast(self):
         self.assertFailIfInactiveSession(gdb_run_cmd(cmd, before=before, target=target))
         res = gdb_run_silent_cmd(cmd, before=before, target=target)
         self.assertNoException(res)
-        self.assertIn("Fastbins[idx=0, size=0x20]", res)
+        if IS_64B:
```

Comment:
in general can we write tests for 32 bit in the `else`? I don't like 32 bit being tested less.